routes/web.php

```diff
@@ -38,6 +38,15 @@
     return view('dashboard');
 })->middleware(['auth'])->name('dashboard');
 
+require __DIR__ . '/auth.php';
+
+Route::get('/overons', function() {
```

Comment:
Zou je dit via een controller willen doen in plaats van de view returnen in de route. Dit is misschien op het moment niet belangrijk, maar om de code uitbreidbaar te houden is het wel handig.